roles/openshift_etcd_migrate/library/inline_python.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
```

Comment:
It was used to run the master-config.yaml python code. Removed as it is replace by the yedit.